blustl/milp.py

```diff
@@ -135,4 +135,6 @@ def encode_and_run(phi, g):
         cost = model.objective.value()
         return Result(True, model, cost, sol)
     else:
+        with open('error.milp', 'w') as f:
+            f.write(str(model))
```

Comment:
Can you just put the model with the not implemented error and remove the saving?
(Don't want to do that during an MPC loop)

```python
raise NotImplementedError((status, model))
```